client/index.jsx

```diff
@@ -14,10 +14,8 @@ class App extends React.Component {
     };
   }
 
-
-
   componentDidMount() {
-    axios.get(`http://localhost:3003/${this.state.currentTrailId}/photos`)
+    axios.get(`http://trail-photos-service-dev.us-west-1.elasticbeanstalk.com/${this.state.currentTrailId}/photos`)
```

Comment:
this will need to be a parameterized (variable) either here with a env variable or hoisted to top level of app and passed down. That being said Im getting CORS issues even when pinging from localhost to this: 
![image](https://user-images.githubusercontent.com/4239492/50264190-5bcfc480-03ce-11e9-9a3c-1aef77eabc30.png)

Reply:
Hi @cjm771, I added the variable.  Can you let me know if the CORS issue still exists?